### DISCUSSION TITLE
Position not yet known - Permanent if starting in offline mode

### DISCUSSION THREAD

**Discussion by bellera** [osmandapp/OsmAnd#11817] [Q&A]
### How to reproduce?
Ensure that OsmAnd is not working.
Put your mobile in air-plane mode.
Launch OsmAnd
Tap the place where should be the Center/GPS button, at the bottom

You will get the message **Position not yet known**. I think it's normal because without connection, no A-GPS data. But the problem is that you remain in this situation for ever!

### My Environment
OsmAnd Version: latest-night-build 25-May-2021
Android/iOS version: 8.1.0
Device model: BQ Aquaris X

<table style="width:100%">
  <tr>
<th><img src="https://user-images.githubusercontent.com/8041180/120336581-c8b34c80-c2f2-11eb-9569-86dd5ef05085.jpg"  width="300" height="auto"/> </td>
  </tr>
</table>

**Comment by vshcherb** [comment]
- Without A-GPS it could take long time to get GPS-signal. Please enable GPS-Status and see how many satellites are connected to your device. So it should increase from 0... up to 10. 
- Another idea that GPS is turned off automatically, once you switch to Airplane mode. So you need to enable it manually.

P.S. I don't think it's related to navigation cause there is no GPS. And you can test it by waiting once Location button becomes blue (i.e. signal is found)

**Reply by bellera** [reply]
>     * Without A-GPS it could take long time to get GPS-signal. Please enable GPS-Status and see how many satellites are connected to your device. So it should increase from 0... up to 10.

I know. It always remains to 0/##.
 
>     * Another idea that GPS is turned off automatically, once you switch to Airplane mode. So you need to enable it manually.

No. The mobile is showing the location icon (GPS active) at the top of the screen. If I open another app like ViewRanger it takes (logically) some minutes to get the position but it works. Same procedure for OsmAnd doesn't work (for me).

**Reply by vshcherb** [reply]
Very weird... Will test it thoroughly though I've used many times airplane mode especially while flying.

**Comment by jobu22** [comment]
I have the same issue, or even worse issue on Android 5.1. Other applications can access the GPS, but not OsmAnd+(v. 3.9.10). My phone was on flight mode during the update and since then I can't get OsmAnd+ get any GPS signal at all regardless of restarts and re-installations.

**Reply by pebogufi** [reply]
Vshcherb already asked you:
"Please enable GPS-Status and see how many satellites are connected to your device. So it should increase from 0... up to 10."
("GPS Status & Toolbox" is a App !)

**Reply by jobu22** [reply]
GPS Status & Toolbox requires Android 8.0 and up ...
Using other app that works on my android, i get decent signal from more than 10 satellites pretty quickly.

**Comment by Kristjan-Ai** [comment]
I have a similar problem with OsmAnd~ 4.8.6.
Phone: Moto g60s
Android 12
When the app is started **without** global location access turned on (I usually keep it off) and then turned on (in the Android quick settings). Then I have to manually exit and reopen the app to get OsmAnd to register that I turned location access on (to get the location icon to appear in the Android status bar). Otherwise it will give the error "Position not yet known" when the location button is pressed and no location icon will appear in the Android status bar. Navigation with My position also won't start in this case.
Would be great if OsmAnd automatically and periodically checks if location access is now available.
